test/runtests.jl

```diff
@@ -2,7 +2,7 @@ module TestGridapMakie
 
 using GridapMakie
 using Gridap
-using GLMakie
+using CairoMakie
```

Comment:
I see that we are not testing with GLMakie anymore.

Do the tests also pass with GLMakie witht he new compat entries?